pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -86,6 +128,11 @@ def __call__(self, time: float):
                     continue  # pragma: no cover
                 while person.time_of_status_change <= time:
                     person.update_status(person.next_infection_status)
+                    # line directly below is short hand way to check
+                    # infection status is one of a list of options.
+                    # Makes use of the enum
+                    if person.infection_status.value in [3, 4, 5]:
```

Comment:
While using the enum is more concise, its much less readable - can the comment instead say what these enums correspond to/ what this check is for?

Reply:
Switched to use enum names instead of values to fix readability.